### DISCUSSION TITLE
Allow the user to select table they want to load images from before loading images

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5805] [Feature Requests]
> Is there a way to configure the "Attachments" section of S7.8.5 such that it does not automatically load "ALL" images by default upon entering the page?
> 
> We would like the page to load, but not load any images. Then the user picks what they want to load from the picklist.....
> 
> Our Herbarium has 975K+ images and it takes this page some time [minutes] to load. We found that you cannot switch to another section such as Workbench or Queries because it keeps loading the images....

### Important Question
Is the issue that there is a query on how many images there are (counts) that must run before loading images?
If so, then we need to solve this instead.

**Requested By:** Matthew at UMich

**Comment by maxpatiiuk** [comment]
The performance issue is likely not caused by loading the images, but by loading the attachment counts for each table - that makes about 70 queries as soon as you open the page (one query for each attachment)
A performance optimization for this would be to use a group by query (one more reason to add that to the query builder API)
